### ISSUE TITLE
Unable to add Harmony to HA Bridge

### ISSUE THREAD

**Issue by dannieboiz** [bwssytems/ha-bridge#231]
I'm running HA Bridge on a windows PC via task scheduler. I'm able to add my Vera to it and control my lightings from Google Home. I'd like to be able to control my home theater through the harmony hub as well.

On the web portal, I enter my Harmony Hub name/IP address then added email/pw hit save. Page refresh then everything I entered disappeared. I've rebooted and tried many times but it just wouldn't stick. 

Do I need the restful harmony jar running as well?

**Comment by bwssytems** [comment]
Yep, it won't stick when you run it that way as it will not have access to certain file systems on Windows in that mode. I believe there is an issue here in the closed section that may have the solution. I'll try to find it.

In the meantime, if you start it from the command prompt in your own directory, it will work.

**Comment by bwssytems** [comment]
look at issue #144

**Comment by dannieboiz** [comment]
stop HA Bridge, then from command prompt I ran ha-bridge-3.2.2.jar went to the portal added harmony, no go, disabled task scheduler, rebooted, launch my Start HABridge.bat file tried again and still not sticking. Did I do something wrong?

**Comment by bwssytems** [comment]
so in a command prompt have you done "java -jar <location>\ha-bridge-3.2.2.jar"?

**Comment by dannieboiz** [comment]
my script is as follow

_> cd c:\HA Bridge
> java -jar ha-bridge-3.2.2.jar_

and also tried running it in command prompt that way as well. Keep in mind, I don't have a problem with it starting. I'm able to use it without a problem for Vera. I just can't add the harmony remote and hub.

**Comment by bwssytems** [comment]
So, when you bring up a command prompt, it usually puts you into your user directory. Create an habridge directory there and put that jar file. C:\HABRIDGE will be locked down for normal users as it is created under the main drive. Or you have to start it with admin privileges. This is why it does not save.

**Comment by dannieboiz** [comment]
FWIW I run the command/script as with admin privileges 

I tried your process by moving the HA Bridge folder to the user profile folder and ran the jar file from command and is still not able to add. I'm going to reboot and try again but I don't think it would matter.

**Comment by dannieboiz** [comment]
rebooted tried again no luck. :(

**Comment by dannieboiz** [comment]
a little on my progress, I finally got it to add the Harmony Hub

renamed the data folder, restarted HA Bridge then I can add the harmony. compared the rights to all files from new and old and they are identical. 

So I kept the working config file then copied the device.db file over. started HA Bridge again and my devices are still there and working no additional configure needed accepted adding Vera back in. Now I gotta figure out how to work with the Harmony part of things.
